tests/test_main.py

```diff
@@ -119,61 +126,75 @@ def test_pipelines(self, configuration, folder):
                     count = session.scalar(
                         select(func.count(DBResource.hdx_id))
                     )
-                    assert count == 35
+                    check.equal(count, 36)
```

Comment:
These are soft asserts so all asserts will be processed and the results displayed at the end - ie. it will fail after running all tests rather than at the first failed test. This is helpful given the time it takes for the tests to complete

Reply:
That's amazing, love it